connector/fyle_integrations_platform_connector/fyle_integrations_platform_connector.py

```diff
@@ -66,11 +67,12 @@ def set_workspace_id(self):
         self.expense_custom_fields.set_workspace_id(self.workspace_id)
         self.corporate_cards.set_workspace_id(self.workspace_id)
         self.reimbursements.set_workspace_id(self.workspace_id)
+        self.tax_groups.set_workspace_id(self.workspace_id)
 
 
     def import_fyle_dimensions(self):
         """Import Fyle Platform dimension."""
-        apis = ['employees', 'categories', 'projects', 'cost_centers', 'expense_custom_fields', 'corporate_cards']
+        apis = ['employees', 'categories', 'projects', 'cost_centers', 'expense_custom_fields', 'corporate_cards', 'tax_groups']
```

Comment:
we shouldn't do this, we'll end up creating tax groups in intacct and xero where we don't use. Should be doing this optionally.